File: query-dsl-java/src/main/java/com/example/querydsljava/dao/entity/PhoneEntity.java
package com.example.querydsljava.dao.entity;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name = "phone")
public class PhoneEntity {
    @Id
    private String id;
    @Column(name = "name")
    private String name;
    @Column(name = "brand")
    private String brand;
}

File: query-dsl-kotlin/src/main/kotlin/com/example/querydslkotlin/dao/entity/PhoneEntity.kt
package com.example.querydslkotlin.dao.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table


@Entity
@Table(name = "phone")
data class PhoneEntity(
    @Id
    @Column(name="id")
    val id: String,
    @Column(name = "name")
    val name: String,
    @Column (name="brand")
    val brand: String
)
